django/db/migrations/operations/fields.py

```diff
@@ -283,7 +281,7 @@ def state_forwards(self, app_label, state):
                     [self.new_name if n == self.old_name else n for n in together]
                     for together in options[option]
                 ]
-        state.reload_model(app_label, self.model_name_lower, delay=delay)
+        state.reload_model(app_label, self.model_name_lower)
```

Comment:
I changed RenameField instead of AlterField by mistake, but actually RenameField has the same issue (and the other operations as well, perhaps). I'll push a second commit to demonstrate. I'm not sure if it's necessary to remove the delayed rendering entirely. I was hoping you or @MarkusH might be able to advise, else I'll continue digging at this.